### PR TITLE
chore: change Java image order to the newest version as default

### PR DESCRIPTION
**Pull request by lrmtheboss**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [ ] Have you tested your Egg changes?

I have not tested the changes
All that I changed was the images list order such that java 17 is at the top so it's the default. This should help with the newer Minecraft versions and plugin versions that require newer versions of java to run.

* java 17
* java 16
* java 11
* java 8

### CODE REVIEW

File: game_eggs/minecraft/java/mohist/egg-mohist.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2021-06-14T21:28:24+03:00",
+    "exported_at": "2021-12-09T13:31:01-05:00",
```
Comment (Software-Noob) [inline game_eggs/minecraft/java/mohist/egg-mohist.json:7]: nothing was changed here

Comment (lrmtheboss) [reply game_eggs/minecraft/java/mohist/egg-mohist.json]: yeap, sorry. removed the file in the commit below

**Review by Software-Noob — APPROVED** [review]
LGTM, thank you for contributing :+1:

**Comment by PanSzelescik** [comment]
I think this pull request removed Mohist egg :P
https://github.com/parkervcp/eggs/tree/master/game_eggs/minecraft/java/mohist

**Comment by Software-Noob** [comment]
> I think this pull request removed Mohist egg :P [`master`/game_eggs/minecraft/java/mohist](https://github.com/parkervcp/eggs/tree/master/game_eggs/minecraft/java/mohist?rgh-link-date=2021-12-14T19%3A08%3A35Z)

You're right. Oops.

**Pull request merged**